torch/csrc/PyInterpreter.cpp

```diff
@@ -770,16 +894,10 @@ c10::SymIntArrayRef ConcretePyInterpreterVTable::sym_strides(
   TORCH_CHECK(
       py::isinstance<py::tuple>(out) || py::isinstance<py::list>(out),
       "sym_strides must be a list or a tuple");
-  size_t len = py::len(out);
-  c10::SymInt* ptr = new c10::SymInt[len];
-  auto capsule = py::capsule(
-      ptr, [](void* p) { delete[] reinterpret_cast<c10::SymInt*>(p); });
-  int64_t idx = 0;
-  for (auto it = out.begin(); it != out.end(); ++it, ++idx) {
-    ptr[idx] = py::cast<c10::SymInt>(*it);
-  }
-  set_tensor_attr_with_capsule(self, capsule, "_sym_strides_capsule");
```

Comment:
Don't forget to remove this func :)

Reply:
I'm actually still using this func (we still need to set a pycapsule buffer on the tensor)

Reply:
Ah whoops!